### PR TITLE
Update 21CSR208.txt

### PR DESCRIPTION
**Pull request by Sriram-Suresh-S**
Adding dept

### CODE REVIEW

File: 21CSR208.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Name : Sriram
 Roll No : 21CSR208
+Dept : CSE
```
Comment (SriRam-3) [inline 21CSR208.txt:3]: what reason?

Comment (Sriram-Suresh-S) [reply 21CSR208.txt:3]: for adding dept.

Comment (SriRam-3) [reply 21CSR208.txt:3]: ok

**Review by SriRam-3 — APPROVED** [review]
approved

**Pull request merged**